### DISCUSSION TITLE
When trying to @mention somebody in a discussion thread only past contributors show up on mobile

### DISCUSSION THREAD

**Discussion by ghost** [community/community#2904] [Mobile] [answered]
When I try to @ mention somebody in a discussion thread, only past contributors to the repo show up in the quick selection list. There's not the person who made the thread, or anyone who participated unless they are past contributors. 

If what I'm trying to reference to doesn't make sense, I mean this list:

![image](https://user-images.githubusercontent.com/65474198/101670783-db6a5a00-3a10-11eb-99db-6b3fb26da3d4.png)

Also, you can only scroll under the extra line shown there that seems out of place.

**Comment by candyho** [comment, marked as answer]
Hey thanks for flagging! I've filed an issue in our internal repo to track the @mention and the UI bugs, will keep you posted once they are fixed.
